modules/lambda_function/lambda-function.md

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Why is this file repeated?

Reply:
Remember the duplication thing Sachin is taking care of? This needs to be here until that is merged